ledger/participant-integration-api/src/main/scala/platform/store/backend/common/UserManagementStorageBackendTemplate.scala

```diff
@@ -132,7 +132,6 @@ object UserManagementStorageBackendTemplate extends UserManagementStorageBackend
             $grantedAt
          )
          """.executeUpdate()(connection)
-    rowsUpdated == 1
```

Comment:
Changing to `Unit` as previously was either true or unique constraint violation error.